### PR TITLE
Add Scroll Back to top button

### LINKED ISSUE #168: [Feature] Add a scroll back to top button
## Problem
There is no scroll back to top button in the CC Resource Archive page. 

## Description
I want to add a scroll back to top button at the bottom of the CC Resource Archive page. In case the number of resources is high and the user scrolls down much, this feature will help them to scroll back to top easily.

## Implementation

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by piyushs131**
## Fixes

- Fixes #168 by @riddhi9570

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by possumbilities** [comment]
Hi @piyushs131 This is for an Issue that has not yet moved to `status: ready for work`. 

Please see the [Contribution Guidelines](https://opensource.creativecommons.org/contributing-code/).

**Comment by possumbilities** [comment]
Hi @piyushs131, thanks for this. There's some changes that would need to be made

* this links to an externally hosted resource, and that's not going to work. I'd strongly suggest you write out vanilla js local to this repository
* there shouldn't be changes to the `Gemfile.lock` file
* an anchor with an `href="#"` isn't adequately accessible and should target an actual anchor that's defined at the top of the page.

**Comment by Murdock9803** [comment]
Hello @piyushs131,
It is always great to include good descriptive text in your PR, which helps others to understand the motive and technical changes proposed by you

**Comment by piyushs131** [comment]
@possumbilities sure, l will go ahead and refactor the code again and submit the PR soon. Thanks for your clarification.

**Comment by Murdock9803** [comment]
@piyushs131 Let me know if you need any help with this.

**Comment by possumbilities** [comment]
@piyushs131 are you still working through the changes here?

**Comment by piyushs131** [comment]
@possumbilities i apologies for the delay due to my other commitments. I assure you, that i will do all necessary changes which you mention and  submit the PR promptly.

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -270,6 +270,7 @@ GEM
 PLATFORMS
   aarch64-linux
   arm64-darwin-23
+  x86_64-linux
```
Comment (Murdock9803) [inline docs/Gemfile.lock:273]: Appears this file is additional here. If its presence is not intentional or necessary, it should be considered for removal to maintain the cleanliness and efficiency of the file.

**Pull request closed without merging**